src/main/java/org/sonarlint/intellij/ui/ruledescription/RuleHeaderPanel.kt

```diff
@@ -115,7 +116,7 @@ class RuleHeaderPanel(private val parent: Disposable) : JBPanel<RuleHeaderPanel>
         updateCommonFields(type, issue.getCleanCodeAttribute(), issue.getImpacts(), issue.getRuleKey())
         updateRuleSeverity(severity)
 
-        if (issue is LiveIssue && canBeReopened(project, issue)) {
+        if (issue is LiveIssue && canBeReopened(project, issue) || issue is LocalTaintVulnerability && canBeReopened(project, issue)) {
```

Comment:
Maybe move all checks in `canBeReopened`?

Reply:
Actually just `canBeReopened(project, issue)` is needed as issue is by default an `Issue`